docs/changelog/changelog.rst

```diff
@@ -54,17 +54,21 @@ Changes
   the environment to exit rather than fail silently.
   This includes all relevant methods in the
   :class:`~holodeck.weather.WeatherController`.
+- Removed the ability to toggle sensors during runtime with the removal of
+  :class:`~holodeck.environments.command.SetSensorEnabledCommand`,
+  :meth:`~holodeck.environments.HolodeckEnvironment.set_sensor_enabled`, and
+  :meth:`~holodeck.sensors.HolodeckSensor.set_sensor_enable`.
```

Comment:
Add a note saying they can make a custom configuration without a sensor if they don't want to use it

Reply:
@jaydenmilne does this work?